### ISSUE TITLE
libman.so

### ISSUE THREAD

**Issue by MaksymMalin** [asmadsen/react-native-unity-view#15]
Hello still got libma.so error though i have alredy checked Player settings a it was mentioned inprevious answers   `java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.xyz--xcZJ-kO3KCWgDTEKcmJxQ==/base.apk"],nativeLibraryDirectories=[/data/app/com.xyz--xcZJ-kO3KCWgDTEKcmJxQ==/lib/x86, /data/app/com.xyz--xcZJ-kO3KCWgDTEKcmJxQ==/base.apk!/lib/x86, /system/lib]]] couldn't find "libmain.so"`

**Comment by MaksymMalin** [comment]
Is it workable on real device . I cannot see if emulator run it/.

**Comment by luojiong** [comment]
@asmadsen 
Hi, can you take some time to reply to this question? This library is very valuable to our users

**Comment by MaksymMalin** [comment]
Yep. I have tested on real device and it works but still cannot launch it on emulator.

**Comment by luojiong** [comment]
Hi, what version of 'react native' you are using? I can't start it on the physical device now, and it will crash when I enter 'app'. I use 'react native 0.62.2' and 'unity' version is' 2019.3.5 '

**Comment by MaksymMalin** [comment]
"dependencies": {
    "@asmadsen/react-native-unity-view": "0.0.6",
    "expo": "~38.0.1",
    "expo-splash-screen": "~0.4.0",
    "expo-status-bar": "^1.0.0",
    "expo-updates": "~0.2.10",
    "react": "~16.11.0",
    "react-dom": "~16.11.0",
    "react-native": "~0.62.2",
    "react-native-gesture-handler": "~1.6.1",
    "react-native-reanimated": "~1.9.0",
    "react-native-screens": "~2.9.0",
    "react-native-unimodules": "~0.10.1",
    "react-native-web": "~0.11.7"

**Comment by MaksymMalin** [comment]
2019.4.f1

**Comment by ostryhub** [comment]
Unity only exports fo arm7 and arm64 architectures. The android emulator running on the desktop pc is x86 (usually). So it is impossible to get the unity exported project to run on android emulator.
